.github/workflows/reuse-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: 2020 Free Software Foundation Europe e.V.
+# SPDX-License-Identifier: CC0-1.0
+name: REUSE Compliance Check
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: REUSE Compliance Check
+        uses: fsfe/reuse-action@v4
+  
```

Comment:
There's still a newline error here, please fix it

Reply:
Also, this workflow is not related to the `Tidy up old copyright info` commit it's in